cogs/Admin.py

```diff
@@ -0,0 +1,154 @@
+            Showing the top **{}** players.
+            Card Count Filter Type: **{}**
+        '''.format(n, card_type_arg)),
+        colour=discord.Colour.blue()
+    )
+
+    embed.set_author(
+        name=clan_info['name'],
+        icon_url='https://icon-library.net//images/clash-royale-icon/clash-royale-icon-8.jpg'
+    )
+
+    rank_values = []
+    for idx, member in enumerate(top_n):
+        mc = member['card_level_counts']
+        rank_text_row = '` {:2d} ` | ` ({:2d}, {:2d}), {:2d}, {:2d} ` | **{}**'.format(
+            idx + 1, mc[14], mc[13], mc[12], mc[11], member['name']
+        )
+        rank_values.append(rank_text_row)
+    field_name = '**Rank** | **# of lvl (14, 13), 12, 11** | **Player Name**'
+    embed.add_field(name=field_name, value='\n'.join(rank_values), inline=False)
+
+    return embed
+
+def boat_attackers_embed(boat_attackers, clan_info):
+    if len(boat_attackers) == 0:
+        description = 'There are no clan members who attacked enemy boats this week.'
+        colour = discord.Colour.blue()
+    else:
+        description = f'**{len(boat_attackers)}** player(s) attacked enemy boats this week.'
+        colour = discord.Colour.orange()
+
+    embed = discord.Embed(description=description, colour=colour)
+    embed.set_author(name=clan_info['name'], icon_url=clan_info['logo_url'])
+
+    if len(boat_attackers) != 0:
+        columns = 'Boat attacks | Medals | Name'
+        row_values = []
+        for p in boat_attackers:
+            row_val = '`     {:^2}    `|`  {:^4} `| {}'.format(
+                p[5], p[6], p[1]
+            )
+            row_values.append(row_val)
+        embed.add_field(name=columns, value='\n'.join(row_values), inline=False)
+
+    return embed
+
+class Admin(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+        self.clan_members_ranker = ClanMembersRanker()
+        self.royale_api_website_scraper = RoyaleApiWebsiteScraper()
+
+    async def fetch_ranked_members(self, ctx, clan_tag, card_type_arg='all'):
+        async with ctx.typing():
+            logging.info("Started fetch ranked members processing...")
+            clan_info, clan_members_ranked = self.clan_members_ranker.get_clan_cards_rank(
+                clan_tag, card_type_arg)
+            logging.info("Completed fetch ranked members processing")
+
+        await ctx.send(embed=create_clan_members_ranked_embed(clan_info, clan_members_ranked, card_type_arg))
+
+    async def card_type_check(self, ctx, card_type):
+        valid_arguments = schemas.CARD_TYPE_ID_PREFIX.values()
+        if card_type not in valid_arguments:
+            message = 'Card Filter parameter is not in scope, please provide correct card filter: troops, spells, or buildings'
+            await ctx.send(embed=discord.Embed(
+                description=message,
+                colour=discord.Colour.red()
+            ))
+            return False
+        else:
+            return True

```

Comment:
/nit please add a new line after this. And/Or perhaps just run `make lint` and fix as many lint warnings as you can